TrackletGraph.py

```diff
@@ -744,8 +755,8 @@ def add_upstream_proc_modules(aMemModule, ProcList=[], MemList=[]):
         if aMemModule is None:
             return ProcList, MemList
 
-        # A few special cases: stop further expanding them upstream
-        if aMemModule.mtype in ['VMStubsME','AllStubs','TrackletParameters','AllProj']:
```

Comment:
@aryd Without `'AllStubs'` in this list and the similar list below in the `add_downstream_proc_modules` method, the generation of the IRtoTB chain (both with non-combined and with combined modules) fails with the following:
```
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│                                                                                                  │
│ /home/hart/Research/CMS/tracklets/firmware-hls/emData/project_generation_scripts/./generator_hdl │
│ .py:508 in <module>                                                                              │
│                                                                                                  │
│   505 │                                                                                          │
│   506 │   for mem in memory_list:                                                                │
│   507 │   │   # Get widths of all needed memories                                                │
│ ❱ 508 │   │   TrackletGraph.populate_bitwidths(mem,args.hls_dir)                                 │
│   509 │   │   TrackletGraph.populate_is_binned(mem,args.hls_dir)                                 │
│   510 │   │                                                                                      │
│   511 │   │   # Determine which memories need numEntries output ports                            │
│ /home/hart/Research/CMS/tracklets/firmware-hls/emData/project_generation_scripts/TrackletGraph.p │
│ y:240 in populate_bitwidths                                                                      │
│                                                                                                  │
│    237 │   │   elif mem.mtype == "DiskStubWord":                                                 │
│    238 │   │   │   mem.bitwidth = 49                                                             │
│    239 │   │   else:                                                                             │
│ ❱  240 │   │   │   raise ValueError("Bitwidth undefined for "+mem.mtype)                         │
│    241 │   │                                                                                     │
│    242 │   │   # Populate BX bit width                                                           │
│    243 │   │   if (      mem.mtype == "TrackletProjections" or mem.mtype == "VMProjections"      │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
ValueError: Bitwidth undefined for CleanTrack
```

If I restore `'AllStubs'`, then it seems to produce correct results in both cases. Was there a reason for removing `'AllStubs'`?